File: pdlearning/src/web-apps/src/libraries/domain-api/src/form/models/form-question-answer-statistics.model.ts
export interface IFormQuestionAnswerStatisticsModel {
  answerCode: number;
  answerValue: string;
  answerCount: number;
  answerPercentage: number;
}

export class FormQuestionAnswerStatisticsModel implements IFormQuestionAnswerStatisticsModel {
  public answerCode: number;
  public answerValue: string;
  public answerCount: number;
  public answerPercentage: number;
}

File: pdlearning/src/microservices/lnaform-microservice/Api/Application/Models/FormQuestionAnswerStatisticsModel.cs
using System;
using Microservice.LnaForm.Domain.Entities;

namespace Microservice.LnaForm.Application.Models
{
    public class FormQuestionAnswerStatisticsModel
    {
        public FormQuestionAnswerStatisticsModel()
        {
            // Do not remove this constructor. It is being used by JsonSerializer.Deserialize
        }

        public int AnswerCode { get; set; }

        public string AnswerValue { get; set; }

        public double AnswerCount { get; set; }

        public double AnswerPercentage { get; set; }
    }
}
